File: app/src/main/java/com/neome/api/meta/base/dto/StudioFieldPinShape.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.EnumDefnMapPinShape;
import com.neome.api.meta.base.Types.MetaIdField;
import com.neome.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioFieldPinShape extends StudioFieldEditable
{
  @Nullable
  public MetaIdField defaultFieldId;

  @Nullable
  public EnumDefnMapPinShape defaultValue;

  @Nullable
  public MetaIdVar defaultVarId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioFieldPinShape.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnMapPinShape
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioFieldEditable

open class StudioFieldPinShape : StudioFieldEditable()
{
  var defaultFieldId: MetaIdField? = null
  var defaultValue: EnumDefnMapPinShape? = null
  var defaultVarId: MetaIdVar? = null
}